src/inference/src/threading/ie_cpu_streams_executor.cpp

```diff
@@ -98,7 +98,12 @@ struct CPUStreamsExecutor::Impl {
                     // wrapping around total_streams (i.e. how many streams all different core types can handle
                     // together)
                     const auto total_streams = _impl->total_streams_on_core_types.back().second;
+                    const auto hybrid_core = _impl->total_streams_on_core_types.size() > 1;
                     const auto streamId_wrapped = _streamId % total_streams;
+                    const auto use_logic = _impl->_config._threads_per_stream_big * _impl->_config._big_core_streams >
+                                           _impl->_config._small_core_offset / 2;
+                    const auto phy_core_streams = _impl->_config._big_core_streams / 2;
+                    const auto cur_logic = use_logic ? (streamId_wrapped >= phy_core_streams ? true : false) : false;
```

Comment:
This logic hard to understand. Could you please add the comments with explanation why we have to different paths for cpu_idx_offset and stream_id computation

Reply:
sure. will add comments of this logic in pinning explanation.